.github/workflows/main.yml

```diff
@@ -340,7 +340,7 @@ jobs:
     if: ${{ needs.semantic-release.outputs.release == 'True' }}
     runs-on: ubuntu-latest
     env:
-      EM_VERSION: 3.1.26
```

Comment:
maybe this can be set to latest as in the test job? Or there was a particular reason to stick with this specific version?

Reply:
ah no ok I see what you mean. Yes it is so that it doesn't breaks when we have no time to review the upgrade.

Reply:
Fort testing is going to be okay. For release we should have a way to reproduce builds in case of bugs. You don't want to release zenroom-js version without knowing the exact build tools. Nice observation, though!